source/manual/fall-back-to-mirror.html.md

```diff
@@ -1,90 +1,83 @@
 
-Every day at 20:00, the [govuk_seed_crawler][] on the Mirrorer machine adds all GOV.UK URLs listed
-within the sitemaps (in the magnitude of hundreds of thousands) to a message queue. The [govuk_crawler_worker][]
-on the Mirrorer machine consumes these URLs, retrieves the HTML returned by these URLs, saves the content
-to disk and adds any new URLs found on those pages to the back of the queue.
+Then the [`govuk_crawler_worker`](https://github.com/alphagov/govuk_crawler_worker) on the mirrorer machine:
 
-Every hour, the static copy of the site is copied from the Mirrorer machine to the primary AWS S3 bucket
-`govuk-<environment>-mirror`. This primary bucket is automatically replicated to another S3 bucket
-(named `govuk-<environment>-mirror-replica`) in another region by AWS.
+- consumes the fetched GOV.UK URLs
+- retrieves the GOV.UK HTML files returned by these URLs
+- saves these HTML files to a `/tmp` folder on the mirrorer machine
+- adds any new URLs found on those pages to the back of the message queue
 
-In addition, the primary AWS S3 bucket `govuk-<environment>-mirror` is synced to the Google GCS bucket
-of the same name daily at 12:00.
+Every hour, the [`govuk_sync_mirror`](https://github.com/alphagov/govuk-puppet/blob/86d1480c6e081313c415246063d5931af24473da/modules/govuk_crawler/manifests/init.pp#L109) script runs on the mirrorer machine. This script copies the GOV.UK HTML files from the mirrorer machine to the main `govuk-<environment>-mirror` AWS S3 bucket. AWS then copies this main bucket to the replica `govuk-<environment>-mirror-replica` S3 bucket in another region.
 
-The crawler is entirely independent of the mirrors. Stopping the crawler means no new updates are made
-to the mirrors, but it will not stop the mirrors from working.
+Finally, every day at 6:00pm, a job is run in Google Cloud Storage (GCS) which syncs the primary AWS S3 bucket `govuk-<environment>-mirror` to the GCS `govuk-<environment>-mirror` bucket. For more information on GCS, see the [Google Cloud Platform documentation](https://docs.publishing.service.gov.uk/manual/google-cloud-platform-gcp.html).
 
-To inspect the contents of the mirror:
+The `govuk_seed_crawler` and `govuk_crawler_worker` scripts are independent of the mirrors. Stopping these scripts stops the mirror updates, but does not stop the mirrors from working.
 
-```bash
+Run the following to inspect the contents of the GOV.UK mirrorer machine:
+
+```
 gds govuk connect -e production ssh mirrorer
 cd /mnt/crawler_worker/www.gov.uk
 ```
 
+## “Mirror GOV.UK content to S3” alert
+
+If the govuk_sync_mirror cronjob has not succeeded for 24 hours, it triggers the “Mirror GOV.UK content to S3” alert. See the [Mirror GOV.UK content to S3 alert documentation](/manual/alerts/mirror-sync.html) for more information.
```

Comment:
```suggestion
If the `govuk_sync_mirror` cronjob has not succeeded for 24 hours, it triggers the “Mirror GOV.UK content to S3” alert. See the [Mirror GOV.UK content to S3 alert documentation](/manual/alerts/mirror-sync.html) for more information.
```

Reply:
Whoops! thank you, will add to next commit